src/subdomain/subdomain.controller.ts

```
@@ -0,0 +1,20 @@
+import { Response } from 'express';
+import { SubdomainService } from './subdomain.service';
+
+@Controller('subdomain')
+export class SubdomainController {
+  constructor(private readonly subdomainService: SubdomainService) {}
+
+  @Post('setup')
+  async setupSubdomain(@Body() body: any, @Res() res: Response) {
+    const { subdomain, port, dnsRecordId } = body;
+    try {
+      const result = await this.subdomainService.setupSubdomain(subdomain, port, dnsRecordId);
```

ESLint

[error] Replace `subdomain,·port,·dnsRecordId` with `⏎········subdomain,⏎········port,⏎········dnsRecordId⏎······`

(prettier/prettier)